cf_xarray/accessor.py

```diff
@@ -430,13 +456,21 @@ def _build_docstring(func):
     can be used for arguments.
     """
 
+    get_all_docstring = (
+        f"One or more of {(_AXIS_NAMES + _COORD_NAMES)!r};\n\t\t\tor standard names"
+    )
     # this list will need to be updated any time a new mapper is added
     mapper_docstrings = {
         _get_axis_coord: f"One or more of {(_AXIS_NAMES + _COORD_NAMES)!r}",
         _get_axis_coord_single: f"One of {(_AXIS_NAMES + _COORD_NAMES)!r}",
         _get_groupby_time_accessor: "Time variable accessor e.g. 'T.month'",
         _get_with_standard_name: "Standard names",
         _get_measure_variable: f"One of {_CELL_MEASURES!r}",
+        _single(_get_all): get_all_docstring.replace("One or more of", "One of"),
```

Comment:
With the decorator this doesn't work well... Looks like anytime I use a decorator the function gets a different id, so the key `_single(_get_all)` is not in ``_DEFAULT_KEY_MAPPERS``. This is the only workaround I can think of: use `func.__name__` as keys of this dictionary, and force the decorator to add an attribute/annotation so we know that `_single` was applied. When we loop later using ``_DEFAULT_KEY_MAPPERS``, we replace "One or more of" with "One of" if there is the decorator attribute/annotation. There must be a cleaner way...

Reply:
we could add these docstrings to the actual functions. and `_single` could extract `__doc__`...

This was me trying to be fancy so we could also just delete it. Up to you...

Reply:
I think `__doc__` works well